modules/platforms/cpp/ignite/client/table/table.h

```diff
@@ -63,14 +63,27 @@ class table {
      *
      * @return Record binary view.
      */
-    [[nodiscard]] IGNITE_API record_view<ignite_tuple> record_binary_view() const noexcept;
+    [[nodiscard]] IGNITE_API record_view<ignite_tuple> get_record_binary_view() const noexcept;
```

Comment:
We have `get_` prefix here, but no prefix for `name()` above - is that right? What is the convention?